docs/en/Getting-Started.md

```diff
@@ -22,25 +22,16 @@ The guide will cover the following tasks:
 [AWS account with an access key id and secret access key](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSGettingStartedGuide/AWSCredentials.html).
 The optional sections will be labeled with **[Bonus]**.
 
-**Note**  This guide is designed to start an Alluxio system with minimal setup. Alluxio
-performs best in a distributed environment for big data workloads, but this scenario is difficult
-to simulate on a single machine. The performance benefits of Alluxio are illustrated in
-the following whitepapers, which include further instructions for running Alluxio in a scaled-up
-environment:
-* [Accelerating on-demand data analytics with Alluxio](https://alluxio.com/resources/accelerating-on-demand-data-analytics-with-alluxio)
-* [Accelerating data analytics on Ceph object storage with Alluxio](https://www.alluxio.com/blog/accelerating-data-analytics-on-ceph-object-storage-with-alluxio).
+**Note**  This guide is designed to start an Alluxio system with minimal setup on a single machine.
+To try Alluxio a distributed environment for big data workloads, request [a sandbox cluster](https://www.alluxio.org/sandbox/request) with Alluxio and Spark installed on AWS for free.
 
 ## Prerequisites
 
-* Mac OS X or Linux
+* MacOS X or Linux
```

Comment:
Technically, nowadays it's just "MacOS" they've dropped the X. The last "X" release was El Capitan in 2015

https://en.wikipedia.org/wiki/MacOS#OS_X_10.11_El_Capitan